functors_task_1/student/include/impl.h

```diff
@@ -8,6 +8,18 @@
  */
 struct BeerOrganizer
 {
+    BeerOrganizer(): currentBear{BeerBrand::None} {}
+
+    BeerBrand operator()() {
+        if (currentBear == BeerBrand::Leffe) {
```

Comment:
please avoid magic values, specify range "from" and "to" with constexpr variables

Reply:
Done